src/components/layers-panel/layers-panel.tsx

```diff
@@ -143,17 +146,29 @@ const HorizontalLine = styled.div`
   opacity: 0.12;
 `;
 
-const getLayerExamples = () => Object.values(EXAMPLES);
+const InsertPanelWrapper = styled.div`
+  position: absolute;
+  top: 24px;
+  // Make insert panel centered related to layers panel.
+  // 168px is half insert panel width.
+  left: calc(50% - 168px);
+`;
+
+const getLayerExamples = (): LayerExample[] => Object.values(EXAMPLES);
```

Comment:
I'm not sure if we still need to store `EXAMPLES` as Object. Possibly Array would be better.

Reply:
Do it in another PR